README.md

```diff
@@ -115,17 +117,37 @@ foundryup
 curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
 ```
 
+### Installing Colima
+
+```bash
+brew install colima
+colima start
+```
+
+### Installing Docker
+
+```bash
+brew install docker
+```
+
 ## Setup and Execution
 
+You can run the AVS both locally using `anvil` or on Holesky testnet.
+
 A `Makefile` is provided to ease the execution. Follow these steps in separate terminals:
 
-1. Build contracts: `make build-contracts`
-2. Start local blockchain: `make anvil`
-3. Deploy contracts: `make deploy-contracts`
-4. Launch first Operator: `make run-operator-uji`
-5. Launch second Operator: `make run-operator-floki`
-6. Start Aggregator: `make run-aggregator`
-7. Create a test task: `make create-task`
+### Local Execution
+
+1. Start local blockchain: `make anvil`
+2. Launch first Operator: `make run-operator-uji-anvil`
+3. Launch second Operator: `make run-operator-floki-anvil`
+4. Start Aggregator: `make run-aggregator-anvil`
+5. Create a test task: `make create-task-anvil`
+
+### Holesky Testnet
+
+1. Launch first Operator: `make run-operator-uji-holesky`
+2. Launch second Operator: `make run-operator-floki-holesky`
+3. Start Aggregator: `make run-aggregator-holesky`
+4. Create a test task: `make create-task-holesky`
 
```

Comment:
One too many blanks at the end of the file here

Reply:
[Remove blank line](https://github.com/gizatechxyz/avsthon/pull/11/commits/fe1146f366fe4464c7a99ee03c4db53d74ed1ceb)